practice/caesar/site.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**Returns the ASCII position of the inputted character.
+*/
+function getIndex(letter) {
+  return letter.charCodeAt();
+}
+
+/** Returns the ASCII character as a string associated with supplied index.
+*/
+function getLetter(index) {
+  return String.fromCharCode(index);
+}
+
+/** Takes in a string and shifts it [key] amount on the alphabet to encrypt
+the word.
+*/
+function caesarEncrypt(key, inputWord) {
+  var wordAsArray = inputWord.split('');
+  var indexArray = _.map(wordAsArray, getIndex);
+  var shiftedIndexArray = _.map(indexArray, function addKey(index) {
+    if (index === 32) {
+      return index;
+    } else {
+      var newIndex = index + key;
+    }
+    if (newIndex > 122) {
+      newIndex -= 26;
+    } else if (newIndex > 90 && newIndex < 97) {
+      newIndex -= 26;
+    }
+    return newIndex;
+  });
```

Comment:
All of these numbers feel like "magic numbers". If you put them in constants, then it will annotate what they mean.